sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuJsonTuple.scala

```diff
@@ -84,25 +90,24 @@ case class GpuJsonTuple(children: Seq[Expression]) extends GpuGenerator
       outer: Boolean,
       targetSizeBytes: Long,
       maxRows: Int = Int.MaxValue): Array[Int] = {
+
     val inputRows = inputBatch.numRows
     // if the number of input rows is 1 or less, cannot split
     if (inputRows <= 1) return Array()
-    val outputRows = inputRows
 
-    val jsonGpuExpr = GpuBindReferences.bindReference(jsonExpr, 
-      Seq(jsonExpr.asInstanceOf[NamedExpression].toAttribute))
+    val outputRows = inputRows
+    val json = inputBatch.column(generatorOffset).asInstanceOf[GpuColumnVector].getBase.incRefCount
```

Comment:
Same question about why incrementing refcount just to close it -- we should just use it without incrementing or closing.

Reply:
Thank you very much! I updated the changes accordingly.